File: 01_Language/01_Functions/python/func_get_args.py
# coding: utf-8

def foo(*args, **kwargs):
    print("Number of arguments: ", len(args))
    if len(args) >= 2:
        print("Second argument is: ", args[1])

    for k, v in enumerate(args):
        print("Argument %d is: " % k, v)


foo(1, 2, 3)

File: 01_Language/01_Functions/js/func_get_args.js
function foo() {
    let args = arguments
    console.log("Number of arguments: ", args.length);
    if (args.length >= 2) {
        console.log("Second argument is: ", args[1]);
    }

    for (let i in args) {
        console.log(`Argument ${i} is: `, args[i]);
    }
}

foo(1, 2, 3);